Changes

```diff
@@ -2,6 +2,14 @@
 Revision history for Log-Any-Adapter-DERIV
 {{$NEXT}}
 
+0.005     2023-08-24 09:54:52+00:00 UTC
```

Comment:
I guess it will be released as version 0.006 ? we should add something under {{$NEXT}}
```suggestion
{{$NEXT}}
    - something changed

0.005     2023-08-24 09:54:52+00:00 UTC
```

Reply:
added